### PR TITLE
Change docstring generation for operations to enable operation decorators

### PR DESCRIPTION
**Pull request by analogue**
Pretty self explanatory.  This is needed so that `bravado.cliient.CallableOperation` can be wrapped by YAD (yet another decorator).

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+5.0.0 (2015-XX-XX)
+---------------------
+- Remove pythonic docstring generation for operations so that they can be further decorated.
+  Instead of `help(petstore.pet.getPetById)` do `print petstore.pet.getPetById.docstring`.
```
Comment (prat0318) [inline CHANGELOG.rst]: So will `petstore.pet.getPetById?` not work from now. That's not good :(

Comment (analogue) [inline CHANGELOG.rst]: Yeah, I'd have to agree with that.

File: bravado/docstring_property.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            return self.fget(obj)
+
+    def __set__(self, obj, value):
+        raise AttributeError("can't set attribute")
+
+    def __delete__(self, obj):
+        raise AttributeError("can't delete attribute")
+
+
+if __name__ == "__main__":
```
Comment (prat0318) [inline bravado/docstring_property.py]: I dont think we will need this `main` section.

**Comment by prat0318** [comment]
:+1:

**Pull request merged**